tests/_test_msui/test_mscolab.py

```diff
@@ -411,6 +411,74 @@ def assert_logout_text():
         qtbot.wait_until(assert_label_text)
         # ToDo verify all operations disabled again without a visual check
 
+    def test_multiple_flightpath_switching_to_flighttrack_and_logout(self, qtbot):
+        """
+        checks that we can switch in topviews with the multiple flightpath dockingwidget
+        between local flight track and operations, and we are able to cycle a login/logout
+        """
+        # more operations for the user
+        for op_name in ["second", "third"]:
+            assert add_operation(op_name, "description")
+            assert add_user_to_operation(path=op_name, emailid=self.userdata[0])
+
+        self._connect_to_mscolab(qtbot)
+        modify_config_file({"MSS_auth": {self.url: self.userdata[0]}})
+        self._login(qtbot, emailid=self.userdata[0], password=self.userdata[2])
+
+        # test after activating operation
+        self._activate_operation_at_index(0)
+        self.window.actionTopView.trigger()
+
+        def assert_active_views():
+            # check 1 view opened
+            assert len(self.window.get_active_views()) == 1
+        qtbot.wait_until(assert_active_views)
+        topview_0 = self.window.listViews.item(0)
+        assert topview_0.window.tv_window_exists is True
+        topview_0.window.setAttribute(QtCore.Qt.WA_DeleteOnClose)
+
+        def assert_attribute():
+            assert topview_0.window.testAttribute(QtCore.Qt.WA_DeleteOnClose)
+        qtbot.wait_until(assert_attribute)
```

Comment:
I am pretty sure that setAttribute is an immediate action that does not require this kind of wait, but I wouldn't know how to find out if this is definitely the case.